x/concentrated-liquidity/tick_test.go

```diff
@@ -409,40 +409,100 @@ func (s *KeeperTestSuite) TestGetTickInfo() {
 func (s *KeeperTestSuite) TestCrossTick() {
 	var (
 		preInitializedTickIndex = DefaultCurrTick.Int64() - 2
+		expectedUptimes         = getExpectedUptimes()
+		emptyUptimeTrackers     = wrapUptimeTrackers(expectedUptimes.emptyExpectedAccumValues)
 		defaultAdditiveFee      = sdk.NewDecCoinFromDec(USDC, sdk.NewDec(1000))
 	)
 
 	tests := []struct {
 		name                         string
 		poolToGet                    uint64
+		preInitializedTickIndex      int64
 		tickToGet                    int64
+		initGlobalUptimeAccumValues  []sdk.DecCoins
+		globalUptimeAccumDelta       []sdk.DecCoins
+		expectedUptimeTrackers       []model.UptimeTracker
 		additiveFee                  sdk.DecCoin
 		expectedLiquidityDelta       sdk.Dec
 		expectedTickFeeGrowthOutside sdk.DecCoins
 		expectedErr                  bool
 	}{
 		{
-			name:                         "Get tick info on existing pool and existing tick",
+			name:                    "Get tick info on existing pool and existing tick below current tick (nonzero uptime trackers)",
+			poolToGet:               validPoolId,
+			preInitializedTickIndex: preInitializedTickIndex,
+			tickToGet:               preInitializedTickIndex,
+			additiveFee:             defaultAdditiveFee,
+			// Global uptime accums remain unchanged after tick init
+			initGlobalUptimeAccumValues: expectedUptimes.twoHundredTokensMultiDenom,
+			globalUptimeAccumDelta:      expectedUptimes.hundredTokensMultiDenom,
+			// We expect new uptime trackers to be new global - init global
+			// This is because we init them to twoHundredTokensMultiDenom and then add hundredTokensMultiDenom,
+			// so when we cross the tick and "flip" it, we expect threeHundredTokensMultiDenom - twoHundredTokensMultiDenom
```

Comment:
Does "flip it" mean one side of the tick is cross so the liquidity is flipped to one token?

Reply:
It means switching a tick's uptime growth "outside" from `growth above the tick` to `growth below the tick` (or vice versa)